BiometricLocker/BiometricLocker.swift

```diff
@@ -99,7 +112,7 @@ open class BiometricLocker {
                 if success {
                     self.delegate?.didAuthenticateSuccessfully()
                     self.positiveFeedbackGenerator.impactOccurred()
-
+                    self.unlock()
```

Comment:
i think maybe these should have `weak self`

Reply:
👌🏽